In [1]:
import math

If the International Space Station is at an altitude of 385 km, that is our starting point, in other words it is a y-intercept because we would be 385 km at h = 0, where h is our input variable. 

The question is asking how many hours it will take for the International Space Station to reach a star that is 4.8 lightyears away. To do this, we need to first convert 4.8 lightyears to kilometers so we are in a similar unit as the ISS's starting altitude. We also need to know the rate of travel which is given to us as 27,707 km per hour. Using this information, we can create a linear equation as follows:

new Altitude = Rate of Travel per Hour * Number of Hours + Starting Altitude 

We'll represent new Altitude as y, rate of travel per hour as m, number of hours as h, and starting altitude as b.

$$ y = mx + b $$ 

m = 27,707 km per hour, b = 385 km, the new altitude we are trying to approach is 4.8 lightyears which we can say is our y 

The first step is to change the rate of travel into scientific notation:

$$ 27,707 = 2.7707 \times 10^4 $$ 

The next step is to change 385 km to scientific notation:

$$ 385 = 3.85 \times 10^2 $$ 

Our y value is given to us in light years, but we need to convert it into kilometers. 

To do this we need to know the conversion rate between the two which is given to us in the problem:

$$ 1 light year = 9.5 \times 10^{12} km $$ 

If we are trying to get to a star that is 4.8 light years away from our current position, we need to convert 4.8 light years into km. 

4.8 light years = 

$$ 4.8 * 9.5 \times 10^{12} $$ 

In [21]:
4.8 * 9.5 

45.6

We need to reshape the product into scientific notation by making it:

$$ 45.6 = 4.56 \times 10^{1} $$

We can then put this exponent of 1 to be added to the 12.

$$ 4.56 \times 10^{13} $$ 

Now that we have 4.8 light years in kilometers, we can subtract the starting altitude and then divide by the rate of travel to get the number of hours needed to reach the star.

$$ 